tests/e2e/credential-webhook/00-step.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl create ns cred-test
```

You can use `apply` field instead of calling `kubectl apply` for every manifest.
More information [here](https://kuttl.dev/docs/testing/reference.html#teststep).

---

this wouldn't let us ignore errors which we need in scenario of testing webhooks